File: week1/Banknotes.py
n = int(input())
vals = [100, 50, 20, 10, 5, 2 , 1]

print(n)
for val in vals:
    tot = n // val
    n += -(tot*val)
    print(f"{tot} nota(s) de R$ {val},00" )

File: week1/banknotes.java
package week1;

import java.util.Scanner;

public class banknotes {

    public static void main(String[] args) {
        int[] notes = {100,50,20,10,5,2,1};
        Scanner scan = new Scanner(System.in);

        int n = scan.nextInt();
        System.out.println(n);
        for (int note:
            notes ) {
            int tot = n/note;
            n -= tot*note;
            System.out.printf("%d nota(s) de R$ %d,00\n",tot,note);
        }
    }


}
